app/src/main/res/values/strings.xml

```diff
@@ -108,4 +108,14 @@
     <string name="edit_backpack_item_error_quantity">Please provide a quantity</string>
     <string name="edit_backpack_item_error_quantity_value">"Please enter a valid number quantity!"</string>
 
+    <!-- Group -->
+    <string name="group_create_title">Create a group</string>
+    <string name="group_create_info_1">În eventualitatea unui cutremur este important sa ai un protocol împreună cu cei apropiați prin care vă puteți regăsi cu ușurință. Creează un grup de prieteni, de familie sau de colegi de la birou sau școală pentru a vă seta puncte de întâlnire în caz de cutremur.</string>
+    <string name="group_create_info_2">Chiar dacă nu vei avea acces la internet în acele momente, poți folosi în continuare aplicația pentru a vedea care sunt punctele de întâlnire și pentru a identifica traseele pentru a ajunge acolo. De asemenea, poți vedea, cu permisiunea lor, traseele și adresele uzuale ale membrilor grupului tău și îi poți căuta dacă aceștia nu vin la punctul de întâlnire.</string>
```

Comment:
please add these also in the RO strings file :)

Reply:
Do we have these texts already translated into English? Or should I just use the Romanian version in both the `strings` and `strings.RO` file?

Reply:
we don't have the final texts / translations, we'll add them when we're closer to finishing the app :D but it's easier to have them in both strings files so we won't forget to adjust them ;)